code/controllers/subsystem/tickets/mentor_tickets.dm

```diff
@@ -21,10 +21,10 @@ GLOBAL_REAL(SSmentor_tickets, /datum/controller/subsystem/tickets/mentor_tickets
 	db_save_id = "MENTOR"
 
 /datum/controller/subsystem/tickets/mentor_tickets/Initialize()
+	..()
```

Comment:
why bother calling parent here if you're just going to override `close_messages` ?

Reply:
These should call parent a-la `atom/Initialize`.

Even if we override it, child SSs should do what their parents do.

Reply:
Makes sense 👍